CHANGELOG.md

```
@@ -2,6 +2,23 @@
 
 ## [Unreleased]
 
+### Added
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)